docs/source/contributing/developing.adoc

```diff
@@ -316,6 +316,13 @@ The following options are available:
 
 Tags::
 Use `tags` to ensure that scenarios and features containing at least one of the selected tags are executed.
+To select particular feature you can use its name as a tag.
+For example, `cmd-config.feature` contains `@cmd-config` tag through which the feature file can be selected and run: `make integration GODOG_OPTS=--tags=cmd-config`.
+There are also tags covering specific areas and usages of {project}:
+
+- The `command` tag indicates features which focus on testing of particular subcommands, for example `cmd-addons.feature` 
```

Comment:
This sentence is confusing. What about "The `command` tag indicates what is being tested of the particular sub command, for example `cmd-addons.feature".

Reply:
'To select a particular feature, you can use ...'